packages/web/components/Modal/CreateInstitutionsModal/index.js

```diff
@@ -0,0 +1,273 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable no-shadow */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import PlacesAutocomplete, { geocodeByPlaceId } from 'react-places-autocomplete';
+import { toast } from '../../Toast';
+import {
+	Form,
+	Actions,
+	InputField,
+	InputHiddenField,
+	MaskedInputField,
+	SelectField,
+	RequiredIndicator,
+} from '../../Form';
+import { InputLabel } from '../../Form/styles';
+import { Container, Button, Suggestion, GoogleAddressSugestions } from './styles';
+import { Cell, Row } from '../../Common';
+import { createInstitutions } from '../../../services';
+import { STATES } from '../../../utils/enums/states.enum';
+import { mapArrayOfObjectToSelect, unMask } from '../../../utils/helper';
+
+const InstitutionsForm = ({ form, closeModal, loading }) => {
```

Comment:
The geolocation select is still not working here and there's a console error saying that this `form` prop is `undefined`.

![image](https://user-images.githubusercontent.com/10481028/101702051-e64bdb80-3a5e-11eb-90dc-4c86765ea51b.png)